CHANGES.md

```diff
@@ -3,9 +3,10 @@ merlin NEXT_VERSION
 
   + merlin binary
     - destruct: Removal of residual patterns (#1737, fixes #1560)
-    - Do not erase fields' names when destructing punned record fields (#1734, 
+    - Do not erase fields' names when destructing punned record fields (#1734,
```

Comment:
I think that `'` after `fields` is a non desired addition (already present before your update)